### DISCUSSION TITLE
Does 2.2.1 Timing Adjustable apply to "silent" server timeouts?

### DISCUSSION THREAD

**Discussion by patrickhlauke** [w3c/wcag#3157] [Q&A]
There are some situations where a session times out server-side, but nothing actively happens (e.g. user is NOT redirected back to login or similar) until the user actually tries to trigger an action or follow a link to another page.

Is this scenario also counted as a failure of 2.2.1 Timing Adjustable (A)? Or is this more of a 2.2.6 Timeouts (AAA)? I've tended towards the former in the past, but the language on https://www.w3.org/WAI/WCAG21/Understanding/timing-adjustable.html doesn't seem to cover this.

/cc @joelamyman

**Comment by JAWS-test** [comment]
What in the SC or Understanding leads you to believe that such a timeout is not covered by 2.2.1?
